pkg/signal/debugtrap.go

```diff
@@ -0,0 +1,18 @@
+// +build !windows
+
+package signal
+
+import (
+	"os"
+	gosignal "os/signal"
+	"syscall"
+)
+
+func setupSigusr1Trap() {
+	c := make(chan os.Signal, 1)
+	gosignal.Notify(c, syscall.SIGUSR1)
+	go func() {
+		<-c
```

Comment:
Wait, no one submitted the requirements document with a specific feature requests to dump **more than once**.  :cry: Embarrassed.. my original code was correct but in all the refactoring I hastily recreated this part and forgot the loop.. fixed now!! :)